temoa/extensions/stochastics/scenario_creator.py

```
@@ -38,10 +40,11 @@ def scenario_creator(scenario_name: str, **kwargs: Any) -> pyo.ConcreteModel:
             # Build a map of table -> index columns from the manifest
             # For each LoadItem, the index columns are all but the last one (which is the value)
-            table_index_map = {}
+            table_index_map: dict[str, list[str]] = {}
+            item: LoadItem
             for item in hybrid_loader.manifest:
                 if item.table not in table_index_map and item.columns:
-                    table_index_map[item.table] = item.columns[:-1]
+                    table_index_map[item.table] = list(item.columns[:-1])
     except Exception as e:
         logger.exception('Failed to connect to database %s', temoa_config.input_database)
         raise RuntimeError(f'Failed to connect to database {temoa_config.input_database}') from e
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)